src/docs/cookbook-manage/implement-delete-user/index.md

```diff
@@ -0,0 +1,574 @@
+
+```shell
+└── src
+    └── containers
+         └── reducers.js
+```
+For more info on the Redux structure, you can read about
+[the Ducks modular Redux proposal](https://github.com/erikras/ducks-modular-redux),
+which is the proposal of Redux usage followed in the FTW templates.
+
+```diff
+  import ContactDetailsPage from './ContactDetailsPage/ContactDetailsPage.duck';
++ import DeleteAccountPage from './DeleteAccountPage/DeleteAccountPage.duck';
+  import EditListingPage from './EditListingPage/EditListingPage.duck';
+  ...
+
+  export {
+    CheckoutPage,
+    ContactDetailsPage,
++   DeleteAccountPage,
+    EditListingPage,
+```
+
+Since DeleteAccountPage has a new route, add the component to the
+application's route configuration.
+
+```shell
+└── src
+    └── routeConfiguration.js
+```
+
+The "Account settings" side navigation tabs are defined in
+routeConfiguration.js as well, so add DeleteAccountPage in the tab
+array.
+
+```diff
+  const ContactDetailsPage = loadable(() => import(/* webpackChunkName: "ContactDetailsPage" */ './containers/ContactDetailsPage/ContactDetailsPage'));
++ const DeleteAccountPage = loadable(() => import(/* webpackChunkName: "DeleteAccountPage" */ './containers/DeleteAccountPage/DeleteAccountPage'));
+  const EditListingPage = loadable(() => import(/* webpackChunkName: "EditListingPage" */ './containers/EditListingPage/EditListingPage'));
+...
+export const ACCOUNT_SETTINGS_PAGES = [
+  'ContactDetailsPage',
+  'PasswordChangePage',
+  'StripePayoutPage',
+  'PaymentMethodsPage',
++ 'DeleteAccountPage',
+];
+...
+      component: PaymentMethodsPage,
+      loadData: pageDataLoadingAPI.PaymentMethodsPage.loadData,
+    },
++   {
++     path: '/account/delete-profile',
++     name: 'DeleteAccountPage',
++     auth: true,
++     authPage: 'LoginPage',
++     component: DeleteAccountPage,
++   },
+    {
+      path: '/terms-of-service',
+```
+
+Finally, add the necessary translation rows into en.js
+
+```shell
+└── src
+    └── translations
+         └── en.js
+```
+
+Feel free to modify the texts to suit your marketplace.
+
+```diff
+  "DateInput.screenReaderInputMessage": "Date input",
++ "DeleteAccountPage.details": "This action is permanent and cannot be undone! After deleting your account, you will be logged out immediately and will not be able to access your listings or transactions anymore.",
++ "DeleteAccountPage.heading": "Delete your account",
++ "DeleteAccountPage.title": "Delete your account",
++ "DeleteAccountPage.error": "Cannot delete user profile. You have {errorCause} Please contact Saunatime support or complete the unfinished issues and try again.",
+  "EditListingAvailabilityForm.fetchMonthDataFailed": "Oops, couldn't load data for {month}, please try again.",
+```
+
+If you now navigate to Account Settings, you can see the "Delete
+account" tab on the left side navigation. If you click the tab, you can
+see the info text.
+
+![Initial Delete Account page](delete-account-page.png)
```

Comment:
https://github.com/sharetribe/flex-docs/pull/545/files#r840489299

My delete account page differed slightly at this point

<img width="963" alt="Screenshot 2022-04-01 at 14 24 34" src="https://user-images.githubusercontent.com/89124964/161254302-9f623463-d22b-4a08-8e5d-f545960af7a4.png">